src/test/java/net/snowflake/client/jdbc/ResultJsonParserV2Test.java

```diff
@@ -223,4 +229,41 @@ public static String stringToHex(String input) {
     }
     return sb.toString().toUpperCase();
   }
+
+  @Test
+  public void testObjectMapperWithLargeJsonString() throws Exception {
+    try {
+      // test with base64 encoded 16M string
+      ObjectMapper mapper = ObjectMapperFactory.getObjectMapper();
+      JsonNode jsonNode = mapper.readTree(generateBase64EncodedJsonString(16 * 1024 * 1024));
+      Assert.assertNotNull(jsonNode);
+      jsonNode = null;
+
+      // test with base64 encoded 32M string
+      ObjectMapperFactory.setMaxJsonStringLength(45000000);
```

Comment:
in `@After` we should set it to the default value

Reply:
Added a cleanup step

Reply:
I don't see the cleanup - where have you added?